independent-publisher-connectors/Zoom Meetings/apiDefinition.swagger.json

```diff
@@ -0,0 +1,430 @@
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "page_size": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "page_size"
+                },
+                "total_records": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "total_records"
+                },
+                "next_page_token": {
+                  "type": "string",
+                  "description": "next_page_token"
+                },
+                "meetings": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "uuid": {
+                        "type": "string",
+                        "description": "uuid"
+                      },
+                      "id": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "id"
+                      },
+                      "host_id": {
+                        "type": "string",
+                        "description": "host_id"
+                      },
+                      "topic": {
+                        "type": "string",
+                        "description": "topic"
+                      },
+                      "type": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "type"
+                      },
+                      "start_time": {
+                        "type": "string",
+                        "description": "start_time"
+                      },
+                      "duration": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "duration"
+                      },
+                      "timezone": {
+                        "type": "string",
+                        "description": "timezone"
+                      },
+                      "created_at": {
+                        "type": "string",
+                        "description": "created_at"
+                      },
+                      "join_url": {
+                        "type": "string",
+                        "description": "join_url"
+                      }
+                    }
+                  },
+                  "description": "meetings"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Meetings",
+        "description": "Gives the Upcoming Meetings",
+        "operationId": "GetMetings",
+        "parameters": []
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}

```

Comment:
@A-Deekshith Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.